### ISSUE TITLE
Notification expecting string but object is passed 

### ISSUE THREAD

**Issue by pvamshi** [AtomLinter/linter-eslint#511]
While fixing errors with `Fix error` option , for some reason the fix failed . But notification received is an object which we are passing as is to atom notification . Which is expecting a string , so it does not show the error message in notification. 
I get the following error in console 

```
Uncaught (in promise) Error: Notification must be created with string message: Error: Linter-ESLint: Fix Attempt Completed, Linting Errors Remain(…)
```

After debugging I realized the `response` object is of the form 

``` js
{
   "message" : "Linter-ESLint: Fix Attempt Completed, Linting Errors Remain"
}
```

The code in `linter-eslint` is like this : 

``` js
        this.worker.request('job', {
          type: 'fix',
          config: atom.config.get('linter-eslint'),
          filePath
        }).then((response) =>
          atom.notifications.addSuccess(response)
        ).catch((response) =>
          atom.notifications.addWarning(response)
        )

```

I am not sure if this something that should be fixed here or else where .

**Comment by Arcanemagus** [comment]
Looks like it should just be changed to `response.message`, right?

**Comment by pvamshi** [comment]
Looks like. But I am not sure if it's in that form all the time . Success message is working fine.  Need more inspection .

**Comment by v3ss0n** [comment]
this making linter broke for my atom, no fix?

**Comment by IanVS** [comment]
@v3ss0n, a PR would be welcome.  Good opportunity towards a hacktoberfest shirt.  😉

**Comment by zaroth** [comment]
It seems I'm still getting a similar error in 2018, on a fresh atom(1.23.3)+eslint(4.15.0)+linter(2.2.0)+linter-eslint(8.4.0) install
![linter-eslint-bug](https://user-images.githubusercontent.com/840021/34863737-4cb2e79e-f771-11e7-9456-978002e61272.gif)
:
Has the bug been reintroduced or is the fix from the PR not released yet?

**Comment by zaroth** [comment]
I wish I could help more, but I can't specify more than knowing that the bugged error message was about me not having correct configuration in the beginning. After some pain and tutorials I managed to set up eslint correctly and the failed notification went away - and all other ones going forward were shown correctly and I can't reproduce the error any more.

I don't really remember what I did that my setup started working - I remember installing `eslint` locally instead of globally, installing `eslint-cli` globally and fiddling around with the `.eslintrc.js` file to try and make it work. As for the project folder, it was a freshly made from `vue-cli`: `vue init webpack-simple`.

**Comment by nauderuan** [comment]
I had the same prob with the right click ESLink Fix, seems the below fixed it for me
![image](https://user-images.githubusercontent.com/1098648/125615874-1bd5c147-2be2-4044-b824-576fbb0eeefb.png)
You might need to disable and enable the eslint plugin for it to take affect

**Comment by LukeKugelmanTW** [comment]
I had the same problem and thanks to @nauderuan comment worked out that my .eslintrc file had a typo and so linter-eslint wasn't finding the config file
